backend/Watchdog.Core/Watchdog.Core.BLL/Services/TeamService.cs

```diff
@@ -0,0 +1,126 @@
+using AutoMapper;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Watchdog.Core.BLL.Services.Abstract;
+using Watchdog.Core.Common.DTO.Team;
+using Watchdog.Core.DAL.Context;
+using Watchdog.Core.DAL.Entities;
+
+namespace Watchdog.Core.BLL.Services
+{
+    public class TeamService : BaseService, ITeamService
+    {
+        public TeamService(WatchdogCoreContext context, IMapper mapper) : base(context, mapper) { }
+        
+        public async Task<ICollection<TeamDto>> GetAllTeamsAsync()
+        {
+            var teams = await _context.Teams
+                .Include(team => team.User)
```

Comment:
Are you sure that we need all this information? 
And maybe it would be better to load teams that belong to the current organization?

Reply:
Removed the loading of unnecessary information.